travis-ymls/ipywidgets.travis.yml

```diff
@@ -2,7 +2,7 @@
 #
 # Package             : ipywidgets
 # Source Repo         : https://github.com/jupyter-widgets/ipywidgets.git
-# Travis Job Link     : https://travis-ci.com/github/dthadi3/ipywidgets/builds/221085037
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/ipywidgets/builds/223543171
```

Comment:
I don't see "tests complete" messages in ppc64le logs unlike amd64. Also I see errors like "Cannot start Firefox" in ppc64le. Pl. check

Reply:
unable to fix the issue... can you please handover this package to your bucket or else assign to anyone.
Thanks in advance.